### PR TITLE
junk field nerf, sorry ya'll.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
Ports: https://github.com/discordia-space/CEV-Eris/pull/7471 

You can no longer release the junkfield due to a gajillion issues caused by spawniing/unspawniing these fields. The weight of 'neutral' fields has been lowered from 10 to 3 which should make you less likely to get a junk field(3:9 ratio of neutral fields to others and 50/50 chance of being an asteroid belt.)

For the record, i did play around with having the field start off instead of on and ultimately couldn't get it working without issues. If by chance you wanna take a crack at it or are away of a solution that Im not please lemme know :p 

</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/adminsay.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@ ADMIN_VERB_ADD(/client/proc/cmd_admin_say, R_ADMIN|R_MOD, TRUE)
 	set category = "Special Verbs"
 	set name = "Asay" //Gave this shit a shorter name so you only have to time out "asay" rather than "admin say" to use it --NeoFite
 	set hidden = 1
-	if(!check_rights(R_ADMIN|R_MOD))	return
+	if(!check_rights(R_ADMIN))	return
```
Comment (SingingSpock) [inline code/modules/admin/verbs/adminsay.dm:7]: Rebase, my dude

Comment (cdb-is-not-good) [reply code/modules/admin/verbs/adminsay.dm:7]: GRAHHHH

Comment (cdb-is-not-good) [inline code/modules/admin/verbs/adminsay.dm:7]:
```suggestion
	if(!check_rights(R_ADMIN|R_MOD)) return
```

File: code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm [hunk 2]
```diff
@@ -43,7 +43,7 @@
 	if(!exploded)
 		exploded = TRUE
 		visible_message(SPAN_DANGER("\the [src] violently explodes!"))
-		explosion(src.loc,0,0,2) //explosion weaker than a welding tank
+		explosion(src.loc, -1, -1, 2, 3) //explosion weaker than a welding tank
```
Comment (cdb-is-not-good) [inline code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm:46]:
```suggestion
		explosion(src.loc,0,0,2) //explosion weaker than a welding tank
```

File: code/game/objects/items/weapons/storage/pouches.dm [hunk 3]
```diff
@@ -285,10 +285,7 @@ obj/item/storage/pouch/large_generic/advmedic/populate_contents()
 		/obj/item/gun/projectile/boltgun/flare_gun,
 		/obj/item/ammo_casing/flare,
 		/obj/item/ammo_casing/shotgun,
-		/obj/item/ammo_magazine/speed_loader_shotgun,
-		/obj/item/ammo_casing/grenade,
-		/obj/item/device/flash,
-		/obj/item/reagent_containers/food/snacks/openable/tastybread
+		/obj/item/ammo_magazine/speed_loader_shotgun
```
Comment (cdb-is-not-good) [inline code/game/objects/items/weapons/storage/pouches.dm:288]:
```suggestion
		/obj/item/ammo_magazine/speed_loader_shotgun,
		/obj/item/ammo_casing/grenade,
		/obj/item/device/flash,
		/obj/item/reagent_containers/food/snacks/openable/tastybread
```

File: code/modules/admin/verbs/adminsay.dm [hunk 4]
```diff
@@ -4,7 +4,7 @@ ADMIN_VERB_ADD(/client/proc/cmd_admin_say, R_ADMIN|R_MOD, TRUE)
 	set category = "Special Verbs"
 	set name = "Asay" //Gave this shit a shorter name so you only have to time out "asay" rather than "admin say" to use it --NeoFite
 	set hidden = 1
-	if(!check_rights(R_ADMIN|R_MOD))	return
+	if(!check_rights(R_ADMIN|R_MOD)) return
```
Comment (cdb-is-not-good) [inline code/modules/admin/verbs/adminsay.dm:7]:
```suggestion
	if(!check_rights(R_ADMIN|R_MOD))	return
```

**Pull request merged**